pulp_smash/tests/rpm/api_v2/test_upload_publish.py

```diff
@@ -19,6 +19,7 @@
     ORPHANS_PATH,
     REPOSITORY_PATH,
     RPM,
+    RPM_DATA,
```

Comment:
Good job seeking out this duplication. There's several other hard-coded references to RPM information littered throughout the code base. They can be taken care of later, though. This is already a definite improvement.